### PR TITLE
feat: trending view ui improve

### PR DESCRIPTION
**Pull request by Lanttcat**
## Description

Closes #MF-474

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/mask/shared-ui/locales/en-US.json [hunk 1]
```diff
@@ -382,10 +383,13 @@
     "plugin_trader_safety_alert": "Anyone can create and name any ERC20 token on Ethereum, including creating fake versions of existing tokens and tokens that claim to represent projects that do not have a token. Similar to Etherscan, this site automatically tracks analytics for all ERC20 tokens independent of token integrity. Please do your own research before interacting with any ERC20 token.",
     "plugin_trader_total_supply": "Total Supply",
     "plugin_trader_circulating_supply": "Circulating Supply",
-    "plugin_trader_volume_24": "Volume (24h)",
+    "plugin_trader_volume_24": "24 Hour Trading Vol",
```
Comment (UncleBill) [inline packages/mask/shared-ui/locales/en-US.json:386]: @xiazhiming Can we align this?

File: packages/mask/src/plugins/Trader/SNSAdaptor/trending/CoinMarketTable.tsx [hunk 2]
```diff
@@ -36,78 +38,74 @@ export function CoinMarketTable(props: CoinMarketTableProps) {
...
                                         {t('plugin_trader_circulating_supply')}
                                     </Typography>
                                 </TableCell>
-                                <TableCell className={classes.head} align="center">
-                                    <Typography color="textSecondary" variant="body2">
-                                        {t('plugin_trader_total_supply')}
-                                    </Typography>
+                                <TableCell className={classes.cell}>
+                                    {trending.market?.circulating_supply
+                                        ? formatSupply(trending.market?.circulating_supply)
```
Comment (UncleBill) [inline packages/mask/src/plugins/Trader/SNSAdaptor/trending/CoinMarketTable.tsx]: Have checked.

```suggestion
                                        ? formatSupply(trending.market.circulating_supply)
```

File: packages/mask/src/plugins/Trader/SNSAdaptor/trending/CoinMarketTable.tsx [hunk 3]
```diff
@@ -36,78 +38,74 @@ export function CoinMarketTable(props: CoinMarketTableProps) {
...
+                                    </Typography>
                                 </TableCell>
-                                <TableCell className={classes.cell} align="center">
-                                    <FormattedCurrency
-                                        value={trending.market?.total_supply ?? 0}
-                                        symbol={trending.coin.symbol}
-                                        formatter={formatCurrency}
-                                    />
+                                <TableCell className={classes.cell}>
+                                    {trending.market?.total_supply ? formatSupply(trending.market?.total_supply) : '--'}
```
Comment (UncleBill) [inline packages/mask/src/plugins/Trader/SNSAdaptor/trending/CoinMarketTable.tsx]:
```suggestion
                                    {trending.market?.total_supply ? formatSupply(trending.market.total_supply) : '--'}
```

File: packages/mask/src/plugins/Trader/SNSAdaptor/trending/TickersTable.tsx [hunk 4]
```diff
@@ -51,15 +55,19 @@ export function TickersTable(props: TickersTableProps) {
     ]
     const tickers = props.tickers.map((ticker, index) => (
         <TableRow key={index}>
-            <TableCell className={classes.cell}>
+            <TableCell className={classes.cell} colSpan={1}>
```
Comment (UncleBill) [inline packages/mask/src/plugins/Trader/SNSAdaptor/trending/TickersTable.tsx]: `1` is the default value, why set it explicitly ?

File: packages/mask/shared-ui/locales/en-US.json (hunk 1 above)
Comment (guanbinrui) [inline packages/mask/shared-ui/locales/en-US.json:386]:
```suggestion
    "plugin_trader_volume_24": "24 Hours Trading Vol",
```

File: packages/mask/src/plugins/Trader/SNSAdaptor/trending/CoinMetadataTable.tsx [hunk 5]
```diff
@@ -25,112 +40,155 @@ const useStyles = makeStyles()((theme) => ({
...
+                                    </TableCell>
+                                </TableRow>
+                            )
+                        })}
+                        {!!trending.coin.community_urls?.length && (
+                            <TableRow>
+                                <TableCell className={classes.cell}>
                                     <Typography className={classes.label} variant="body2">
-                                        {label}
+                                        Community
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Trader/SNSAdaptor/trending/CoinMetadataTable.tsx]: i18n

File: packages/mask/src/plugins/Trader/SNSAdaptor/trending/CoinMetadataTable.tsx [hunk 6]
```diff
@@ -25,112 +40,155 @@ const useStyles = makeStyles()((theme) => ({
...
 }))
 
 export interface CoinMetadataTableProps {
-    trending: Trending
+    trending: TrendingAPI.Trending
     dataProvider: DataProvider
 }
 
+const brands: Record<CommunityType, React.ReactNode> = {
+    facebook: <FacebookRoundIcon sx={{ fontSize: 16 }} />,
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Trader/SNSAdaptor/trending/CoinMetadataTable.tsx:73]: `EnhanceableSite`

File: packages/mask/shared-ui/locales/en-US.json (hunk 1 above)
Comment (UncleBill) [reply packages/mask/shared-ui/locales/en-US.json:386]: `24 hour` is right, according to https://www.quora.com/Is-24-hours-right-or-24-hour

File: packages/mask/src/plugins/Trader/SNSAdaptor/trending/TraderView.tsx [hunk 7]
```diff
@@ -39,56 +40,50 @@ const useStyles = makeStyles<{ isPopper: boolean }>()((theme, props) => {
...
                   borderRadius: 0,
-                  marginBottom: theme.spacing(2),
+                  marginBottom: 0,
               },
         body: props.isPopper
             ? {
                   minHeight: 303,
                   overflow: 'hidden',
-                  border: `solid 1px ${theme.palette.divider}`,
+                  // border: `solid 1px ${theme.palette.divider}`,
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Trader/SNSAdaptor/trending/TraderView.tsx]:
```suggestion
```

File: packages/mask/shared-ui/locales/en-US.json (hunk 1 above)
Comment (guanbinrui) [reply packages/mask/shared-ui/locales/en-US.json:386]: IC

Comment (Lanttcat) [reply packages/mask/shared-ui/locales/en-US.json:386]: Is it used differently elsewhere?

File: packages/mask/src/plugins/Trader/SNSAdaptor/trending/CoinMarketTable.tsx (hunk 3 above)
Comment (Lanttcat) [reply packages/mask/src/plugins/Trader/SNSAdaptor/trending/CoinMarketTable.tsx]: The market maybe undefined
```typescript
export interface Trending {
    currency: Currency
    dataProvider: DataProvider
    coin: Coin
    platform?: Platform
    market?: Market
    tickers: Ticker[]
    lastUpdated: string
}
```

File: packages/mask/shared-ui/locales/en-US.json (hunk 1 above)
Comment (UncleBill) [reply packages/mask/shared-ui/locales/en-US.json:386]: Yes. In the new Trending for NFT, it's still "Volume (24h)"

**Review by UncleBill — DISMISSED** [review]

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**